plugins/generic/googleScholar/GoogleScholarPlugin.inc.php

```diff
@@ -0,0 +1,138 @@
+
+		if (is_a($article, 'PublishedArticle') && ($datePublished = $article->getDatePublished())) {
+			$templateMgr->addHeader('googleScholarDate', '<meta name="citation_date" content="' . strftime('%Y/%m/%d', strtotime($datePublished)) . '"/>');
+		} elseif ($issue && $issue->getYear()) {
+			$templateMgr->addHeader('googleScholarDate', '<meta name="citation_date" content="' . htmlspecialchars($issue->getYear()) . '"/>');
+		} elseif ($issue && ($datePublished = $issue->getDatePublished())) {
+			$templateMgr->addHeader('googleScholarDate', '<meta name="citation_date" content="' . strftime('%Y/%m/%d', strtotime($datePublished)) . '"/>');
+		}
+
+		if ($issue) {
+			$templateMgr->addHeader('googleScholarVolume', '<meta name="citation_volume" content="' . htmlspecialchars($issue->getVolume()) . '"/>');
+			$templateMgr->addHeader('googleScholarNumber', '<meta name="citation_issue" content="' . htmlspecialchars($issue->getNumber()) . '"/>');
+		}
+
+		if ($article->getPages()) {
+			if ($startPage = $article->getStartingPage()) $templateMgr->addHeader('googleScholarStartPage', '<meta name="citation_firstpage" content="' . htmlspecialchars($startPage) . '"/>');
+			if ($endPage = $article->getEndingPage()) $templateMgr->addHeader('googleScholarEndPage', '<meta name="citation_lastpage" content="' . htmlspecialchars($endPage) . '"/>');
+		}
+
+		foreach($templateMgr->get_template_vars('pubIdPlugins') as $pubIdPlugin) {
+			if ($pubId = $pubIdPlugin->getPubId($article)) {
+				$templateMgr->addHeader('googleScholarPubId' . $pubIdPlugin->getPubIdDisplayType(), '<meta name="citation_' . htmlspecialchars(strtolower($pubIdPlugin->getPubIdDisplayType())) . '" content="' . htmlspecialchars($pubId) . '"/>');
+				
+			}
+		}
+
+		$templateMgr->addHeader('googleScholarHtmlUrl', '<meta name="citation_abstract_html_url" content="' . $request->url(null, 'article', 'view', array($article->getBestArticleId($journal))) . '"/>');
+		if ($language = $article->getLanguage()) $templateMgr->addHeader('googleScholarLanguage', '<meta name="citation_language" content="' . htmlspecialchars($language()) . '"/>');
+
+		$i=0;
+		if ($subject = $article->getSubject(null)) foreach ($subject as $locale => $localeSubject) {
+			foreach (explode($localeSubject, '; ') as $gsKeyword) if ($gsKeyword) {
+				$templateMgr->addHeader('googleScholarKeyword' . $i++, '<meta name="citation_keywords" xml:lang="' . htmlspecialchars(substr($locale, 0, 2)) . '" content="' . htmlspecialchars($gsKeyword) . '"/>');
+			}
+		}
+
+		$i=$j=0;
+		if (is_a($article, 'PublishedArticle')) foreach ($article->getGalleys() as $galley) {
+			if (is_a($galley->getFile(), 'SupplementaryFile')) continue;
+				$templateMgr->addHeader('googleScholarPdfUrl' . $i++, '<meta name="citation_pdf_url" content="' . $request->url(null, 'article', 'download', array($article->getBestArticleId($journal), $galley->getBestGalleyId($journal))) . '"/>');
```

Comment:
Just double-checking: the tag here is `citation_pdf_url` but below `citation_fulltext_html_url`? I.e. without `fulltext`?